PublicSDKSwiftAPITester/AttributionNetworkAPI.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  AttributionNetworkAPI.swift
+//
+//  Created by Madeline Beyl on 8/25/21.
+
+import Foundation
+import Purchases
+
+var aNetwork: RCAttributionNetwork!
+func checkAttributionNetworkEnums() {
+    switch aNetwork! {
+    case .appleSearchAds,
+         .adjust,
+         .appsFlyer,
+         .branch,
+         .tenjin,
+         .facebook,
+         .mParticle:
+        print(aNetwork!)
+    @unknown default:
+        fatalError()
```

Comment:
this will work only if we're expecting to run the code, but it won't produce a compile-time build failure. 
I think this isn't ideal, because it's harder to set up a test for it: we'd have to create a test target, and then make sure that other calls _don't_ issue a `fatalError` so we don't get false positives. But this may be hard to do, since we're not passing valid parameters into these calls in any case. 

I think the ideal version of this is something that gets us build failures as soon as we miss a case. If we remove the `@unknown default`, we get a compilation warning, but we get a build failure if we forget a case. I think that's a worthy trade-off, since we probably won't be building the project ourselves - we'll just let CI tell us when we messed up

Reply:
same for all enums